content/add-a-printer-pop.md

```diff
@@ -38,39 +43,42 @@ sudo usermod -aG lpadmin $USER
 
 ## Brother Printers
 
-Search for your printer on this page <https://www.brother-usa.com/brother-support/driver-downloads>
+Brother provides a driver installation tool for Linux users. Install the appropriate driver for your Brother printer by downloading this tool and running the installer with your printer model appended to the command.
 
-Once you find your printer, click the downloads link, and select the "Linux" option under "Select OS Family"
+1. Click [this link](https://support.brother.com/g/b/faqend.aspx?c=us&lang=en&prod=hll2300d_us_eu_as&faqid=faq00100556_000) to download Brother's Driver Install Tool for Linux.
 
-Select Linux (deb) for your "OS Version"
+2. Click the "Click here to download tool" link on the Brother download page.
```

Comment:
Should this be "Click here to download the tool"? I'm seeing the word "the" in the link.

Reply:
Yep, updated